tools/template_based_query_generation/src/test/java/parser/KeywordsMappingTest.java

```diff
@@ -1,48 +1,10 @@
 package parser;
 
-import com.google.common.collect.ImmutableList;
 import org.junit.jupiter.api.Test;
-import token.TokenInfo;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
 
```

Comment:
Please add comments & ToDo when the unit test is not provided.

Reply:
Will add!